node/node.go

```diff
@@ -170,9 +170,15 @@ func NewNode(
 	}
 
 	n := &Node{
-		pubSubEvent:  make(chan net.EvtPubSub),
-		pushLogEvent: make(chan net.EvtReceivedPushLog),
-		peerEvent:    make(chan event.EvtPeerConnectednessChanged),
+		// WARNING: The current usage of these channels means that consumers of them
```

Comment:
That is very true. But it would have been the case before as well but limited to the size of the event bus channel instead of the node channels above.

Reply:
Ah true! That's an interesting thought too, I wonder if that has been affecting things